arguments.py

```diff
@@ -34,11 +55,11 @@ def parse_args() -> argparse.Namespace:
 
     args = parser.parse_args()
     if args.subcommand == "deploy":
-        args.steps = args.steps.split(",")
-        args.skip_steps = args.skip_steps.split(",")
+        args.steps = list(filter(None, args.steps.split(",")))
+        args.skip_steps = list(filter(None, args.skip_steps.split(",")))
```

Comment:
introduce function `remove_empty_strs(..)` and reuse. I only realized what you were doing after reading the commit message. The code is imo easier to read if I don't need to go back to the commit message to understand your intent.